support/mesos-website/Dockerfile

```diff
@@ -13,14 +13,16 @@ USER root
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
       doxygen \
-      locales \
-      ruby \
-      ruby-dev \
-      rubygems && \
```

Comment:
I think with `apt-get` we can also install a specific version of Ruby, right?

Reply:
Right, but the ruby version in the official Ubuntu 16.04 repository are to old.

Reply:
Yes, I think it's reasonable.

We should merge this, and separately see if we can update the docker image to a more recent Ubuntu version, e g. 20.04 or 22.04.

I started working on updating other containers a couple weeks ago, it's not too much work.

Reply:
> and separately see if we can update the docker image to a more recent Ubuntu version, e g. 20.04 or 22.04.
:+1: definitely. Which containers do you already have on your hands? Then I can start to help with others.

Reply:
I started looking at https://hub.docker.com/r/mesos/mesos-tidy and https://hub.docker.com/r/mesos/mesos-build

If you could update this image used for the website build that'd be great, and if you have time maybe also updating the CentOS image https://hub.docker.com/r/mesos/mesos-centos

Thanks!

Reply:
:+1: What do u think about to migrate from centos to almalinux? :thinking:

Reply:
If you guys already started working on updating the images, then do we still need to merge this PR? I mean once the images are updated, we would need to revert the changes of this PR?

Reply:
Good question! Depends the mesos-build image and it's impact. If it's done and all the build jobs are still working,  then it should be very easy to update the "website build" dockerfile. What do u think @cf-natali ? First merge this PR to fix the website?

Reply:
I think it makes sense to merge, this way we're sure the website build is fixed.
Upgrading to a new Ubuntu release might be more work, so I'd rather do it separately, taking the time.